pandas/core/arrays/categorical.py

```diff
@@ -2504,15 +2508,15 @@ def codes(self) -> Series:
         """
         from pandas import Series
 
-        return Series(self._parent.codes, index=self._index)
+        return Series(self._parent.codes, index=self._index, copy=False)
 
     def _delegate_method(self, name, *args, **kwargs):
         from pandas import Series
 
         method = getattr(self._parent, name)
         res = method(*args, **kwargs)
         if res is not None:
-            return Series(res, index=self._index, name=self._name)
+            return Series(res, index=self._index, name=self._name, copy=False)
```

Comment:
We are sure that none of the methods where this is used give a view? (didn't check in detail where this is used, just wondering)

Reply:
I think so, but not 100%. Lets leave this out for now